launcher/ui/dialogs/BlockedModsDialog.cpp

```diff
@@ -186,8 +185,26 @@ void BlockedModsDialog::setupWatch()
 {
     const QString downloadsFolder = APPLICATION->settings()->get("DownloadsDir").toString();
     const QString modsFolder = APPLICATION->settings()->get("CentralModsDir").toString();
-    m_watcher.addPath(downloadsFolder);
-    m_watcher.addPath(modsFolder);
+    watchPath(downloadsFolder, true);
```

Comment:
i don't think this is good default behavior, i wouldn't really feel super well about my MC launcher looking for mods in my MuseScore macOS bundle, were I an average user

![the recursive behavior causes a lot of completely unrelated folders inside the downloads folder to be watched as well](https://user-images.githubusercontent.com/9145768/226763198-c06b8e6a-8037-401a-ab82-03f1b0d2e03b.png)

Reply:
hmm, yeah. this is why I questioned the wisdom of it being default in the issue. I figured I put it out and get comments during review. I'll add a setting.